.editorconfig

```diff
@@ -14,3 +14,6 @@ indent_style = tab
 
 [*.{yaml,yml,conf}]
 indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
why do we need trailing whitespace?

Reply:
to indicate a new line.
what do we usually use?

Reply:
use normal newlines